.ci/StarterRuleSet-AllRulesByCategory.groovy.txt

```diff
@@ -333,7 +333,10 @@ ruleset {
     PackageNameMatchesFilePath
     ParameterName
     PropertyName
-    VariableName
+    VariableName {
+        finalRegex = null
```

Comment:
See https://codenarc.org/codenarc-rules-naming.html#variablename-rule 
From CodeNarc-

> NOTE: Until CodeNarc 2.0, the default naming pattern for final variable names was that they start with an uppercase letter and contain only uppercase letters, numbers and underscores (i.e., like constants). Starting with CodeNarc 2.0, that has been changed so that finalRegex defaults to null and thus final variable names are treated like regular variables. See [#467](https://github.com/CodeNarc/CodeNarc/issues/467). To restore that previous behavior, set finalRegex to [A-Z][A-Z0-9_]*.